src/aero_reps/aero_rep_single_particle_layers.F90

```diff
@@ -632,25 +632,26 @@ end function spec_name
   !> Get the number of instances of a specified aerosol phase. In the single
   !! particle representation, if an aerosol phase is present, is exists once
   !! in each computational particle.
```

Comment:
This description looks like it needs to be updated for the layered particles

Reply:
done!